impeller/renderer/backend/vulkan/android/ahb_texture_source_vk.cc

```diff
@@ -281,6 +281,9 @@ static TextureDescriptor ToTextureDescriptor(
   desc.mip_count = (ahb_desc.usage & AHARDWAREBUFFER_USAGE_GPU_MIPMAP_COMPLETE)
                        ? ahb_size.MipCount()
                        : 1u;
+  if (ahb_desc.usage & AHARDWAREBUFFER_USAGE_COMPOSER_OVERLAY) {
```

Comment:
There is also `AHARDWAREBUFFER_USAGE_GPU_FRAMEBUFFER`. I think this whole method could do a redo to to account for accurate usage. I'll patch it up later.